src/platform/user/authentication/selectors.js

```diff
@@ -0,0 +1,3 @@
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+
+export const ssoe = state => toggleValues(state).ssoe;
```

Comment:
For a bit of extra safety it's best to use the constant to access `ssoe` on the toggleValues, so:

`export const ssoe = state => toggleValues(state)[FEATURE_FLAG_NAMES.ssoe]`

which means you'll have to add this this import up top:

`import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';`

Reply:
good to know, I was [basing it off of this pattern](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/utils/selectors.js#L315-L325).